CONTRIBUTING.md

```diff
@@ -0,0 +1,370 @@
+one commit, often referred to as "squashing" or "rebasing". This is a
+common request by package maintainers when submitting a Pull Request as
+it maintains a more compact commit history. To rebase your commits:
+
+    git rebase -i HEAD~#
+
+Where \# is the number of commits you want to combine. Then you can pick
+the relevant commit message and discard others.
+
+### Pushing your changes
+
+When you want your changes to appear publicly on your GitHub page, push
+your forked feature branch's commits :
+
+    git push origin shiny-new-feature
+
+Here origin is the default name given to your remote repository on
+GitHub. You can see the remote repositories :
+
+    git remote -v
+
+If you added the upstream repository as described above you will see
+something like :
+
+    origin  git://github.com/UV-CDAT/uvcdat.git
+    myuvcdat git@github.com:yourname/uvcdat.git (fetch)
+    myuvcdat git@github.com:yourname/uvcdat.git (fetch)
+
+Now your code is on GitHub, but it is not yet a part of the *UV-CDAT*
+project. For that to happen, a Pull Request needs to be submitted on
+GitHub.
+
+### Review your code
+
+When you're ready to ask for a code review, you will file a Pull
+Request. Before you do, again make sure you've followed all the
+guidelines outlined in this document regarding code style, tests,
+performance tests, and documentation. You should also double check your
+branch changes against the branch it was based off of:
+
+1.  Navigate to your repository on
+    GitHub--<https://github.com/your-user-name/uvcdat>.
+2.  Click on Branches.
+3.  Click on the Compare button for your feature branch.
+4.  Select the base and compare branches, if necessary. This will be
+    master and shiny-new-feature, respectively.
+
+### Finally, make the Pull Request
+
+If everything looks good you are ready to make a Pull Request. A Pull
+Request is how code from a local repository becomes available to the
+GitHub community and can be looked at and eventually merged into the
+master version. This Pull Request and its associated changes will
+eventually be committed to the master branch and available in the next
+release. To submit a Pull Request:
+
+1.  Navigate to your repository on GitHub.
+2.  Click on the Pull Request button.
+3.  You can then click on Commits and Files Changed to make sure
+    everything looks okay one last time.
+4.  Write a description of your changes in the Preview Discussion tab.
+5.  Click Send Pull Request.
+
+This request then appears to the repository maintainers, and they will
+review the code. If you need to make more changes, you can make them in
+your branch, push them to GitHub, and the pull request will be
+automatically updated. Pushing them to GitHub again is done by:
+
+    git push -f myuvcdat shiny-new-feature
+
+This will automatically update your Pull Request with the latest code
+and restart the Travis-CI tests.
+
+### Delete your merged branch (optional)
```

Comment:
Its not from their repo necessarily. It could be deleting a local branch as well.